### PR TITLE
Css js updates

### PR DESCRIPTION
**Pull request by EdwinGuzman**
This fixes:
* #46 and adds other markup and css for a better desktop view.
* #56 for minified css and js, and adds a scss file to compile down to css. The gulp script is part of the python script so that there's only one command to run. I added a few lines in the README about installing it as a dependency.

Other minor a11y and css updates from @ricardoom are here too.

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
And this seems to add a better result in Google's PageSpeed Insights! https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fd210qhkuica97m.cloudfront.net%2F

**Comment by aslagle** [comment]
Could we run gulp from `node_modules/.bin/gulp` so it doesn't require installing gulp globally?

File: about.html.template [hunk 1]
```diff
@@ -35,89 +34,90 @@
...
+        <p>To celebrate cell service and free Wi-Fi in all underground subway stations, NYC’s libraries, MTA, and Transit Wireless have partnered to bring you Subway Library.</p>
+        <p>Simply connect to TransitWireless Wi-Fi in any MTA station. Once you're connected, you can read free books, excerpts, and short stories, all provided by NYC’s public libraries.</p>
+        <p>Subway Library is powered by SimplyE, the free library e-reader app that lets you browse, borrow, and read thousands of library e-books instantly—anywhere, anytime.</p>
 
-          <h3>SimplyE</h3>
-          <img src="iphone_simplyE.png" class="about-simplye" alt="" />
-        </div>
+        <h3>SimplyE</h3>
+        <img src="iphone_simplyE.png" class="about-simplye" alt="" />
+        <img src="iphone_simplyE.png" class="about-simplye desktop" alt="" />
```
Comment (aslagle) [inline about.html.template]: Are these supposed to be the same image?

Comment (EdwinGuzman) [inline about.html.template]: From the designs, yes. Only both appear on desktop view.

**Comment by EdwinGuzman** [comment]
Good point. I tend to run npm modules globally so I usually just leave it that way. Will update.

File: index.scss [hunk 2]
```diff
@@ -604,42 +657,58 @@ footer .sponsors span {
...
+  footer .sponsors span {
+    width: calc(100% * (1/6) - 15px);
+  }
+  footer .sponsors span:first-child,
+  footer .sponsors span:nth-child(2),
+  footer .sponsors span:nth-child(3) {
+    border-bottom: none !important;
+  }
+  footer .sponsors span:nth-child(3) {
+    border-right: 1px solid #DEDEDE !important;
```
Comment (aslagle) [inline index.scss]: why did you need !important for these?

File: about.html.template (hunk 1 above)
Comment (aslagle) [inline about.html.template]: Two copies of the same image appear on desktop view?

Comment (EdwinGuzman) [inline about.html.template]: Correct.

Comment (ricardoom) [inline about.html.template]: we should us the `svg` version of the log here no?

Comment (aslagle) [inline about.html.template]: I also see this image still shows "instant classics" which sadly is no longer in SimplyE. We will need a new image.

Comment (ricardoom) [inline about.html.template]: wherever we have repeated svg elements you can use `<use>` and define the `<svg>` once. 
[<use>](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use)

Comment (EdwinGuzman) [inline about.html.template]: @ricardoom it's the large iPhone image in the about page.

File: index.scss (hunk 2 above)
Comment (EdwinGuzman) [inline index.scss]: I forgot to update that before. It's fixed now to not use `!important`.
